src/VideoFile.ts

```diff
@@ -21,6 +32,13 @@ export interface RecordVideoOptions {
    * Called when the recording has been successfully saved to file.
    */
   onRecordingFinished: (video: VideoFile) => void;
+  /**
+   * IOS only
+   * Specify the video codec to use. To get a list of available video codecs use the `getAvailableVideoCodecs()` function.
```

Comment:
```suggestion
   * Set the video codec to record in. Different video codecs affect video quality and video size.
   * To get a list of all available video codecs use the `getAvailableVideoCodecs()` function.
```

Reply:
This description is wrong, that method isn't actually available, I made a copy-paste error when copying from https://github.com/mrousavy/react-native-vision-camera/pull/2/files#diff-432ccbc3244e65723bf8031a79bbad1191b792c59affdbd2ea93ef6e6b1138d4L6